source/math/Quaternion.ooc

```diff
@@ -36,15 +36,15 @@ Quaternion: cover {
 	// z = this y
 	// w = this z
 
-	inverse ::= This new(this w / (this norm * this norm), -this x / (this norm * this norm), -this y / (this norm * this norm), -this z / (this norm * this norm))
-	isValid ::= (this w == this w && this x == this x && this y == this y && this z == this z)
-	isIdentity ::= (this w == 1.0f && this x == 0.0f && this y == 0.0f && this z == 0.0f)
-	isNull ::= (this w == 0.0f && this x == 0.0f && this y == 0.0f && this z == 0.0f)
-	norm ::= (this real squared() + (this imaginary norm) squared()) sqrt()
-	normalized ::= (this / this norm) as This
+	isValid ::= this real == this real && this imaginary isValid
+	isIdentity ::= this real == 1.f && this imaginary x == 0.f && this imaginary y == 0.f && this imaginary z == 0.f
+	isNull ::= this real == 0.f && this imaginary x == 0.f && this imaginary y == 0.f && this imaginary z == 0.f
+	norm ::= (this real squared() + this imaginary norm squared()) sqrt()
+	normalized ::= this / this norm
 	logarithmImaginaryNorm ::= ((this logarithm) imaginary) norm
```

Comment:
Actually, we should probably use `equals()` when comparing Float.. or? @sebastianbaginski